src/scene/gsplat/shader-generator-gsplat.js

```diff
@@ -16,7 +16,10 @@ const splatCoreVS = /* glsl */ `
 
     varying vec2 texCoord;
     varying vec4 color;
-    varying float id;
+
+    #ifndef DITHER_NONE
```

Comment:
I have not checked the code, but don't we use `id` for the dithering?

Reply:
yep! :)